src/app/pages/waiting-room/waiting-room.page.ts

```diff
@@ -195,6 +196,16 @@ export class WaitingRoomPage extends PracticeableBasePageComponent implements On
     }
   }
 
+  clickBack(): void {
```

Comment:
can you use `async / await` rather than `Promise.then()`?

Reply:
have re-written